### DISCUSSION TITLE
Issue commands when music is playing

### DISCUSSION THREAD

**Discussion by JoaoPedroAssis** [pschatzmann/ESP32-A2DP#280] [Q&A] [answered]
Hi there!

I am using the library to play music while I perform some signal processing tasks in the audio stream. This is one of the tasks that the application will perform. I want to be able to issue commands to the chip to toggle this function on/off, as well as activate other tasks, like in the bellow example:

FLAG = 0 -> play music and process streams
FLAG = 1 -> do other stuff 1
FLAG = 2 -> do other stuff 2

How can I remotely interact with the chip? In the docs its mentioned that A2DP and WiFi cant be used together. Is there another method that can coexist with this library? Thanks in advance

**Comment by pschatzmann** [comment, marked as answer]
I never tried but probably ESP-NOW might work. Usually you would trigger your commands with buttons.
If it needs to be wireless you can add some additional Wifi or Bluetooth module to your solution.

If you have control over the A2DP Source you could try to send AVRC commands.
